### DISCUSSION TITLE
2005/12/09/%E5%91%B3%E4%B9%8B%E9%83%BD%E7%9A%84%E7%9A%AE%E8%9B%8B%E7%98%A6%E8%82%89%E7%B2%A5

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#67] [Announcements]
# 2005/12/09/%E5%91%B3%E4%B9%8B%E9%83%BD%E7%9A%84%E7%9A%AE%E8%9B%8B%E7%98%A6%E8%82%89%E7%B2%A5

因为刚拔了牙，所以中午去公司附近的味之都快餐店点了一份皮蛋瘦肉粥。粥刚端上来的时候我就想，这皮蛋还行，不用嚼也咽得下去，这瘦肉就不能吃了，还得挑出来。结果一砂锅的粥，从头吃到尾，一块瘦肉也没看到，难道他们老板知道我牙齿不好？

https://blog.qizhen.xyz/2005/12/09/%E5%91%B3%E4%B9%8B%E9%83%BD%E7%9A%84%E7%9A%AE%E8%9B%8B%E7%98%A6%E8%82%89%E7%B2%A5

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Emerson
2005年12月10日 23:46 编辑
在 计算机.网络.软件.IM联络聊天 中看到你，就来了。呵呵，您的网页挺强的！！

回复
Unknown
2005年12月9日 13:51 编辑
你是GOOGLE的代言人啊！呵呵
